app/mailers/logbook_entry_mailer.rb

```diff
@@ -1,4 +1,7 @@
 class LogbookEntryMailer < ApplicationMailer
+    has_history
+    track_clicks campaign: "log-book-entry-emails-campaign"
+
     before_action :set_user
     before_action :set_logbook_entry
   
```

Comment:
can remove the `# track_clicks campaign: "logbook_entry_created_email", only: %i[created_email]` comment line thats in this file.